### DISCUSSION TITLE
How to expand / collapse a folder in the Files panel

### DISCUSSION THREAD

**Discussion by lestephane** [jesseduffield/lazygit#4043] [Q&A] [answered]
There is an arrow down icon next to folders, and I've also seen it change to an arrow pointing right, although I don't remember how I did it. I cannot find any documentation on the actual shortcut to collapse (arrow right) and expand (arrow down).

**Comment by mark2185** [comment, marked as answer]
Try `<Enter>`.

Also there's ` if you want to flatten it all.

**Reply by MaxAtoms** [reply]
`Enter` seems to stage all the files within the folder

**Reply by stefanhaller** [reply]
No, it collapses/expands a folder, just like @mark2185 said above.

**Reply by MaxAtoms** [reply]
Oh, you're right. I'm sorry. I confused myself on the Kinesis keyboard where `Space` and `Enter` are right next to each other
